### PR TITLE
Fix kinesis demo

### PR DESCRIPTION
**Pull request by gstroup**
Some dependencies for the kinesis demo were out of date.  I made a few small updates and got it working yesterday.

### CODE REVIEW

File: dataflow-website/recipes/kinesisdemo/kinesisconsumer/pom.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 	<properties>
 		<java.version>1.8</java.version>
-		<spring-cloud.version>Greenwich.SR1</spring-cloud.version>
+		<spring-cloud.version>2020.0.4</spring-cloud.version>
```
Comment (onobc) [inline dataflow-website/recipes/kinesisdemo/kinesisconsumer/pom.xml:18]: This is an older line of Spring Cloud, do you mind moving this up to `2021.0.5`.

**Review by onobc — CHANGES_REQUESTED** [review]
Thanks for the updates @gstroup - nice work!

1 comment to address.

File: dataflow-website/recipes/kinesisdemo/kinesisconsumer/pom.xml (hunk 1 above)
Comment (gstroup) [reply dataflow-website/recipes/kinesisdemo/kinesisconsumer/pom.xml]: done.  :+1:

**Review by onobc — APPROVED** [review]

**Comment by onobc** [comment]
Thanks for the contribution @gstroup !

**Pull request merged**